src/server/detail/save_stages_controller.cc

```diff
@@ -293,11 +292,11 @@ void SaveStagesController::SaveRdb() {
     return;
   }
 
-  auto cb = [snapshot = snapshot.get()](Transaction* t, EngineShard* shard) {
-    snapshot->StartInShard(shard);
-    return OpStatus::OK;
-  };
-  trans_->ScheduleSingleHop(std::move(cb));
+  shard_set->pool()->AwaitFiberOnAll([snapshot = snapshot.get()](util::ProactorBase* pb) {
+    if (auto* shard = EngineShard::tlocal(); shard) {
+      snapshot->StartInShard(shard);
+    }
+  });
 }
```

Comment:
So what do we do with multi-hop operations and multi commands now? We can start serializing non-atomic values? There surely was some reason that SAVE was transctional